### PR TITLE
Original

### PR DESCRIPTION
**Pull request by fractal161**
Been doing a lot of things with the asm lately, and i decided to make some annotations since i'm not familiar with all aspects of nes programming yet. this led to a fair bit of relabeling, where I added some ejona labels and renamed some things on my own too (those should probably be given to taus later). My plan is now to convert all of the LXXX labels into something readable, since I have this vision of the disasm becoming a way to introduce people to romhacking or smth. hopefully you find this interesting :).

(also ignore the makefile i am sorry)

### CODE REVIEW

**Comment by fractal161** [comment]
ah, also forgot to add but i did make sure i didn't break the checksum so the code should be correct at least

**Comment by CelestialAmber** [comment]
Sorry for not looking at this earlier. I'll look at it either today or tommorow.

**Comment by CelestialAmber** [comment]
So first thing, does the makefile not work on Windows as is?

**Comment by fractal161** [comment]
Well I'm running Ubuntu so I wouldn't know :P.

**Comment by CelestialAmber** [comment]
Does it not work on Ubuntu? If so what error does it give?

File: audio/music/music1.asm [hunk 1]
```diff
@@ -1,3 +1,9 @@
+; There are 3 main types of commands, that controller the instrument, duration, and pitch of the note
```
Comment (CelestialAmber) [inline audio/music/music1.asm]: that controls the instrument

**Review by CelestialAmber — CHANGES_REQUESTED** [review]

**Comment by CelestialAmber** [comment]
Sorry for taking so long to look at this. Once you fix the typo and revert your changes on the Makefile I'll merge this.

**Comment by CelestialAmber** [comment]
Also since I don't have a Linux computer, if you could get the Makefile to also work with Linux while not breaking Mac support with Wine I would really appreciate it. You don't have to though ofc.

**Comment by fractal161** [comment]
done.

regarding the makefile, i vaguely recall having some issues with wine in the past, but everything seemed to work just fine now (maybe it was just slow so i changed to local binaries?). i remember that git update-index exists so everything should be fine now.

**Pull request merged**